### PR TITLE
add gradient boosting classifier/regressor tools in sklearn.ensemble

### PR DESCRIPTION
**Pull request by qiagu**

### CODE REVIEW

**Comment by qiagu** [comment]
Finally, the PR is restored.

**Comment by bgruening** [comment]
Cooli! Let me know if you plan to add some tests :)

File: tools/sklearn/ensemble.xml [hunk 1]
```diff
@@ -178,6 +257,27 @@ res.to_csv(path_or_buf = "$outfile_predict", sep="\t", index=False)
...
+            <param name="header2" value="True"/>
+            <param name="col2" value="1"/>
+            <param name="selected_task" value="train"/>
+            <param name="selected_algorithm" value="GradientBoostingRegressor"/>
+            <param name="max_features" value="number_input"/>
+            <param name="num_max_features" value=""/>
+            <param name="random_state" value="42"/>
+            <output name="outfile_fit" file="gbr_model01" compare="sim_size" delta="500"/>
+        </test>
+        <test>
```
Comment (bgruening) [inline tools/sklearn/ensemble.xml]: I guess we also need a test for `GradientBoostingClassifier`?

**Comment by qiagu** [comment]
@bgruening Thank you. Very enjoyed to work with you.
